docs/dev/release.md

```diff
@@ -154,8 +164,18 @@ If script successfully finishes, you should be able to see new tag (e.g. 1.2.0)
 Then go the GitHub and create a PR
 from the `mongodb-fork` repository to https://github.com/redhat-openshift-ecosystem/certified-operators (`origin`).
 
+Before posting the PR there are manual changes you need to make:
+
+1. Ensure to add the `quay.io/` prefix in all Operator image references.
+1. Add missing a `com.redhat.openshift.versions: "v4.8"` line at the end of `metadata/annotations.yaml`.
```

Comment:
```suggestion
1. Add a missing `com.redhat.openshift.versions: "v4.8"` line at the end of `metadata/annotations.yaml`.
```

Reply:
curiosity question: why OpenShift 4.8?

Reply:
Not sure on why that one version in particular, maybe @igor-karpukhin can shed some light here